lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -14,140 +14,158 @@ import LoggedInUser from './logged_in_user';
+  }
+
+  createZendeskTicket(formData) {
+    const url = this.submitFormUrl,
       request = new XMLHttpRequest(),
-      $course = $('#course'),
-      $subject = $('#subject'),
       data = {
         comment: {
-          body: this.state.formData.message,
+          body: formData.message,
         },
+        subject: formData.subject, // Zendesk API requires 'subject'
+        custom_fields: [{
+          id: this.props.context.customFields.course_id,
+          value: formData.course,
+        }],
         tags: this.props.context.tags,
-      },
-      errors = [];
-
-    this.clearErrors();
-
-    data.requester = {
-      email: this.userInformation.email,
-      name: this.userInformation.username,
-    };
-
-    course = $course.find(':selected').val();
-    if (!course) {
-      course = $course.val();
-    }
-    if (!course) {
-      $('#course').closest('.form-group').addClass('has-error');
-      errors.push(gettext('Select a course or select "Not specific to a course" for your support request.'));
-    }
-    data.custom_fields = [{
-      id: this.props.context.customFields.course_id,
-      value: course,
-    }];
-    subject = $subject.find(':selected').val();
-    if (!subject) {
-      subject = $subject.val();
-    }
-    if (!subject) {
-      $subject.closest('.form-group').addClass('has-error');
-      errors.push(gettext('Select a subject for your support request.'));
-    }
-    data.subject = subject; // Zendesk API requires 'subject'
-
-    if (this.validateData(data, errors)) {
-      request.open('POST', url, true);
-      request.setRequestHeader('Content-type', 'application/json;charset=UTF-8');
-      request.setRequestHeader('X-CSRFToken', $.cookie('csrftoken'));
-
-      request.send(JSON.stringify(data));
-
-      request.onreadystatechange = function success() {
-        if (request.readyState === 4 && request.status === 201) {
-          this.setState({
-            success: true,
-          });
-        }
-      }.bind(this);
-
-      request.onerror = function error() {
-        this.setErrorState([gettext('Something went wrong. Please try again later.')]);
-      }.bind(this);
-    }
-  }
-
-  clearErrors() {
-    this.setErrorState([]);
-    $('.form-group').removeClass('has-error');
+        requester: {
+          email: this.userInformation.email,
+          name: this.userInformation.username,
+        },
+      };
+    request.open('POST', url, true);
+    request.setRequestHeader('Content-type', 'application/json;charset=UTF-8');
+    request.setRequestHeader('X-CSRFToken', $.cookie('csrftoken'));
+    request.send(JSON.stringify(data));
+    request.onreadystatechange = function success() {
+      if (request.readyState === 4 && request.status === 201) {
+        this.setState({
+          success: true,
+        });
+      }
+    }.bind(this);
+
+    request.onerror = function error() {
+      this.updateErrorInState('request', this.formValidationErrors.request);
+      this.scrollToTop();
```

Comment:
Why are we scrolling up here?

Reply:
We are scrolling up to the top where all the description of the errors is.